models/folder.go

```diff
@@ -13,6 +13,8 @@ import (
 
 var reSlug = regexp.MustCompile("[^a-zA-Z0-9-]+")
 
+const PostponeByDays = 31
```

Comment:
It's better to use a `time.Duration`: `const PostPonePeriod = 31*24*time.Hour`

And then further down: `ExpiresAt: created.Add(PostponePeriod)`